e2e/testcafe-devextreme/tests/accessibility/drawer.ts

```diff
@@ -0,0 +1,33 @@
+import { Properties } from 'devextreme/ui/drawer.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const options: Options<Properties> = {
+  height: [400],
+  disabled: [true, false],
+  hint: [undefined, 'hint'],
+  revealMode: ['slide', 'expand'],
+  opened: [true, false],
+  position: ['top', 'bottom', 'left', 'right'],
+  template: [() => {
+    const $drawerContent = $('<div>').width(200).css('background-color', 'aqua').css('height', '100%');
```

Comment:
I think we should remove background-color because it could potentially cause an error in the contrast check.